### PR TITLE
chore(release): 4.7.0-alpha.0

### PR DESCRIPTION
**Pull request by Laura-Danielle**
# Overview
Alpha.0 bump for 4.7.0 release.

# Changelog

- updated release notes, let me know if I missed anything.

# Review requests
Please review the release notes.

### CODE REVIEW

File: api/release-notes.md [hunk 1]
```diff
@@ -6,6 +6,19 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
...
+- This is primarily a bug fix release.
+
+## Bug Fixes
+
+- Fixed an issue where pipette offset and tip length calibration would sometimes not be saved correctly.
+- Fixed issues around attach pipette behavior.
+
+---
+
 # OT-2 Software Changes in 4.6.2
```
Comment (mcous) [inline api/release-notes.md]: We might want to demote the headings in this previous release section to match the established conventions of this file

File: app-shell/build/release-notes.md [hunk 2]
```diff
@@ -6,6 +6,20 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
...
+
+None in the Opentrons App.
+
+## Bug Fixes
+
+None in the Opentrons App.
+
+---
+
 # Opentrons App Changes in 4.6.2
```
Comment (mcous) [inline app-shell/build/release-notes.md]: Ditto on previous release heading demotion

**Review by mcous — COMMENTED** [review]
Content of release notes look good to me. Just one comment around heading levels for the prior releases to match the rest of the file

**Review by nusrat813 — APPROVED** [review]
LGTM

**Review by ahiuchingau — APPROVED** [review]

**Review by mcous — APPROVED** [review]

**Pull request merged**